docs/csharp/whats-new/csharp-9.md

```diff
@@ -0,0 +1,233 @@
+
+These features can improve performance in some scenarios. They should be used only after careful benchmarking both before and after adoption. Code involving native sized integers must be tested on multiple target platforms with different integer sizes. The other features require unsafe code.
+
+## Fit and finish features
+
+Many of the other features help you write code more efficiently. In C# 9.0, you can omit the type in a new expression when the created object’s type is already known. The most common use is in field declarations:
+
+:::code language="csharp" source="snippets/whats-new-csharp9/FitAndFinish.cs" ID="WeatherStationField":::
+
+Target type new can also be used when you need to create a new object to pass as a parameter to a method. Consider a `ForecastFor()` method with the following signature:
+
+:::code language="csharp" source="snippets/whats-new-csharp9/FitAndFinish.cs" ID="ForecastSignature":::
+
+You could call it as follows:
+
+:::code language="csharp" source="snippets/whats-new-csharp9/FitAndFinish.cs" ID="TargetTypeNewArgument":::
+
+Another nice use for this feature is to combine it with init only properties to initialize a new object:
+
+:::code language="csharp" source="snippets/whats-new-csharp9/FitAndFinish.cs" ID="InitWeatherStation":::
+
+In addition, you can return an instance created by the default constructor using a `return new();` expression.
+
+A similar feature improves the target type resolution of conditional expressions. With this change, the two expressions need not have an implicit conversion from one to the other, but may both have implicit conversions to a common type. You likely won’t notice this change. What you will notice is that some conditional expressions that previously required casts or wouldn’t compile now just work.
+
+Starting in C# 9.0, you can add the `static` modifier to lambda expressions or anonymous methods. That has the same effect as the `static` modifier on local functions: a static lambda or anonymous function cannot capture local variables or instance state. This prevents accidentally capturing other variables.
+
+Covariant return types enables a function that overrides a base class virtual function to change the return type to a type derived from the type declared in the base class. This can be useful for Records, and for other types that support virtual clone or factory methods.
+
+Next, you can use discards as parameters to lambda expressions. This convenience enables you to avoid naming the argument, and the compiler may be able to avoid using it. You use the `_` for any argument.
```

Comment:
Might be worth mentioning that it was possible to use `_` before for arguments, but you could only use one as it would simple be the name of the argument instance rather than a discard. With this additional functionality you can use more than one `_` and they're true discards, rather than a single variable named `_`.

Reply:
I'm going to leave that out of the what's new. It will be covered in the reference area.